lib/request_started_on_middleware.rb

```diff
@@ -0,0 +1,59 @@
+# Add to config/application.rb:
+#
+#     config.middleware.use 'RequestStartedOnMiddleware'
+#
+class RequestStartedOnMiddleware
+  def initialize(app)
+    @app = app
+  end
+
+  def call(env)
+    begin
+      start_request(env['PATH_INFO'], Time.now.utc)
+      response = @app.call(env)
```

Comment:
This can just be `@app.call(env)` (without the `response` var).  `ensure` doesn't mess with the the result of the block/method:

```console
$ ruby -e "puts begin; 'foo'; ensure; puts 'hi'; end"
hi
foo
```

Reply:
Good point, I'll fix this.